tests/models/store_fs.py

```diff
@@ -228,3 +230,47 @@ def test_save_store_fs_bad_project(tp0_store_fs):
     tp0_store_fs.pootle_path = "/en/project0_BAD/example.po"
     with pytest.raises(ValidationError):
         tp0_store_fs.save()
+
+
+@pytest.mark.django_db
+def test_store_fs_plugin(tp0_store_fs):
+    store_fs = tp0_store_fs
+
+    class DummyPlugin(object):
+
+        file_class = FSFile
+
+        def __init__(self, project):
+            self.project = project
+
+        def foo(self):
+            return "bar"
+
+    @provider(fs_plugins)
+    def provide_plugin(**kwargs):
```

Comment:
this is a dummy version of the FSPlugin.

the provider hooks it up, and lets us test for the file and plugin @property on StoreFS